src/containers/CheckoutPage/CheckoutPage.js

```diff
@@ -230,7 +230,8 @@ export class CheckoutPageComponent extends Component {
       listingId: this.state.pageData.listing.id,
       bookingStart: this.state.pageData.bookingDates.bookingStart,// speculatedTransaction.booking.attributes.start,
       bookingEnd: this.state.pageData.bookingDates.bookingEnd, //speculatedTransaction.booking.attributes.end,
-      quantity: this.state.pageData.bookingData.quantity
+      quantity: this.state.pageData.bookingData.quantity,
+      test: "test"
```

Comment:
what is this test?

Reply:
this I used for testing if a new param will pass through, I will remove it now